v3/lints/cabf_br/lint_cab_dv_conflicts_with_street.go

```diff
@@ -25,6 +22,20 @@ import (
 
 type certPolicyConflictsWithStreet struct{}
 
+// If the Certificate asserts the policy identifier of 2.23.140.1.2.1, then it MUST NOT include
+// organizationName, streetAddress, localityName, stateOrProvinceName, or postalCode in the Subject field.
+
+func init() {
+	lint.RegisterLint(&lint.Lint{
+		Name:          "e_cab_dv_conflicts_with_street",
+		Description:   "If certificate policy 2.23.140.1.2.1 (CA/B BR domain validated) is included, streetAddress MUST NOT be included in subject",
+		Citation:      "BRs: 7.1.6.1",
```

Comment:
A quick question as there seem to be a bunch of these '.1'->'.4' changes, is this a case of the citation in the code being out of date and not reflecting changes made to the cabf br document since this lint was written or were they wrong to start with and are finally being fixed now?

Reply:
They were changed in https://cabforum.org/2020/09/09/ballot-sc35-cleanups-and-clarifications/ in response to CA confusion about the intersection of the requirements and subscriber certs vs CA certs.

Reply:
Oh, and the existing lints already checked if they were not-CA certs, so the lints were correctly scoped to only subscriber certs, but the requirement was moved from 7.1.6.1 (all certs) to 7.1.6.4 (subscriber certs)